specs/windows/signature.table

```diff
@@ -0,0 +1,10 @@
+table_name("signature")
+description("File (executable, bundle, installer, disk) code signing status.")
+schema([
+    Column("path", TEXT, "Must provide a path or directory", required=True),
+    Column("result", TEXT, "The signature check result")
+])
+implementation("signature@generateQueryResults")
+examples([
+  "select * from signature where path = 'C:\\Windows\\notepad.exe'",
```

Comment:
Would you mind adding in an example that joins off of the process table just to show up the extra cool features? :D

Reply:
Done! I've added a simple example in the table specs, and also edited the PR summary to include a couple of more (that we can probably add to the docs once this is merged).